src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConversionTests.cs

```diff
@@ -1214,5 +1214,49 @@ static void M(decimal? d)
 ";
             CompileAndVerify(source).VerifyDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(54113, "https://github.com/dotnet/roslyn/issues/54113")]
+        public void DisallowedModreqOnConversion()
```

Comment:
>DisallowedModreqOnConversion [](http://example.com/codeflow?start=20&length=28)

This looks like dupe of the other test that was already added to the other test file.

Reply:
I wanted to have a test that covered a modreq, as opposed to covering `UnmanagedCallersOnly`. Previous to this change, this test would not have reported any errors.

Reply:
> I wanted to have a test that covered a modreq

Sorry, missed that.